packages/synapse-react-client/src/components/ChallengeRegisterButton/ChallengeRegisterButton.tsx

```diff
@@ -135,17 +100,19 @@ const ChallengeRegisterButton = ({
         <SpinnerButton
           disableElevation={true}
           variant="contained"
-          color="secondary"
-          onClick={() => (onJoinClick ? onJoinClick() : undefined)}
+          onClick={() =>
+            accessToken && onJoinClick ? onJoinClick() : undefined
+          }
           sx={{
             color: 'white',
+            backgroundColor: '#109488',
```

Comment:
Quick and dirty background color override since we can't use the secondary version of the button here either.

Reply:
In Storybook it looks like we can just add the `color="secondary"` prop. Why can't we do that here?

Reply:
If we set the secondary color to this in the challenge portal, it changes the colors of many other elements that we don't want to change